models/tmp/stg_apple_search_ads__search_term_report_tmp.sql

```diff
@@ -1,3 +1,3 @@
-{{ config(enabled=var('apple_search_ads__using_search_terms', True)) }}
+{{ config(enabled=var('apple_search_ads__using_search_terms', False)) }}
 
 select * from {{ var('search_term_report') }}
```

Comment:
Small style request to adjust the `tmp` folder models to have the same structure. It looks like they vary across the tmp models.